content/en/docs/Writing policies/variables.md

```diff
@@ -351,6 +377,48 @@ Reference the image properties of initContainer `vault`:
 
 `{{images.initContainers.vault.digest}}`
 
+6. Reference the readable reference for the image (along with the image digest)
+
+`{{images.initContainers.vault.reference}}`
+
+7. Reference the readable reference for the image (along with the image tag)
+
+`{{images.initContainers.vault.referenceWithTag}}`
+
+For initContainer `vault`, `referenceWithTag` corresponds to `https://ghcr.io/vault:v3` since the image was specified with a tag and not a digest. The `reference` variable will also hold the same value.
+
+Reference the image properties of container `nginx`:
+
+1. Reference the registry URL
+
+`{{images.containers.nginx.registry}}`
+
+2. Reference the path to the image
+
+`{{images.containers.nginx.path}}`
+
+3. Reference the image name
+
+`{{images.containers.nginx.name}}`
+
+4. Reference the image tag
+
+`{{images.containers.nginx.tag}}`
+
+5. Reference the digest
+
+`{{images.containers.nginx.digest}}`
+
+6. Reference the readable reference for the image (along with the image digest)
+
+`{{images.containers.nginx.reference}}`
+
+7. Reference the readable reference for the image (along with the image tag)
+
+`{{images.containers.nginx.referenceWithTag}}`
+
+For the container `nginx`, `referenceWithTag` corresponds to `https://docker.io/library/nginx:`. Since the image was specified with a digest, `reference` corresponds to `https://docker.io/library/nginx@sha256:abcdef1234567890`.
+
```

Comment:
Again, this is redundant. Just one for `images.containers.` and one for `images.initContainers`.